lib/api/gitlab.js

```diff
@@ -0,0 +1,296 @@
+  createPr,
+  getPr,
+  updatePr,
+  // file
+  commitFileToBranch,
+  getFile,
+  getFileContent,
+  getFileJson,
+};
+
+// Initialize GitLab by getting base branch
+async function initRepo(repoName, token) {
+  logger.debug(`initRepo(${repoName})`);
+  if (token) {
+    process.env.GITLAB_TOKEN = token;
+  } else if (!process.env.GITLAB_TOKEN) {
+    throw new Error(`No token found for GitHub repository ${repoName}`);
+  }
+  config.repoName = repoName.replace('/', '%2F');
+  try {
+    const res = await glGot(`projects/${config.repoName}`);
+    config.defaultBranch = res.body.default_branch;
+    logger.debug(`${repoName} default branch = ${config.defaultBranch}`);
+  } catch (err) {
+    logger.error(`GitHub init error: ${JSON.stringify(err)}`);
+    throw err;
+  }
+}
+
+// Search
+
+// Returns an array of file paths in current repo matching the fileName
+async function findFilePaths(fileName) {
+  logger.verbose('Can\'t find multiple package.json files in GitLab');
+  return [fileName];
+}
+
+// Branch
+
+// Returns true if branch exists, otherwise false
+async function branchExists(branchName) {
+  logger.debug(`Checking if branch exists: ${branchName}`);
+  try {
+    const url = `projects/${config.repoName}/repository/branches/${branchName}`;
+    const res = await glGot(url);
+    if (res.statusCode === 200) {
+      logger.debug('Branch exists');
+      return true;
+    }
+    // This probably shouldn't happen
+    logger.debug('Branch doesn\'t exist');
+    return false;
+  } catch (error) {
+    if (error.statusCode === 404) {
+      // If file not found, then return false
+      logger.debug('Branch doesn\'t exist');
+      return false;
+    }
+    // Propagate if it's any other error
+    throw error;
+  }
+}
+
+// Returns the Pull Request for a branch. Null if not exists.
+async function getBranchPr(branchName) {
+  logger.debug(`getBranchPr(${branchName})`);
+  const urlString = `projects/${config.repoName}/merge_requests?state=opened`;
+  const res = await glGot(urlString);
+  logger.debug(`Got res with ${res.body.length} results`);
+  let pr = null;
+  res.body.forEach((result) => {
+    if (result.source_branch === branchName) {
+      pr = result;
+    }
+  });
+  if (!pr) {
+    return null;
+  }
+  return getPr(pr.id);
+}
+
+// Issue
+
+async function addAssignees(prNo, assignees) {
+  logger.debug(`Adding assignees ${assignees} to #${prNo}`);
+  if (assignees.length > 1) {
+    logger.error('Cannot assign more than one assignee to Merge Requests');
+  }
+  let url = `projects/${config.repoName}/merge_requests/prNo`;
+  url = `${url}?assignee_id=${assignees[0]}`;
+  await glGot.put(url);
+}
+
+async function addReviewers(prNo, reviewers) {
+  logger.debug(`addReviewers('${prNo}, '${reviewers})`);
+  logger.error('No reviewer functionality in GitLab');
+}
+
+async function addLabels(prNo, labels) {
+  logger.debug(`Adding labels ${labels} to #${prNo}`);
+  let url = `projects/${config.repoName}/merge_requests/prNo`;
```

Comment:
I realized what the error was I was receiving. The `prNo` string needs to be changed to `${prNo}`.